### DISCUSSION TITLE
Flashing a new FS (as uf2) on the Rpi Pico W

### DISCUSSION THREAD

**Discussion by adaptiveaurash** [micropython/micropython#13643] [RP2040 / Pico]
Hi Guys,

I'm trying to flash a new FS on Rpi Pico  (as a very quick method to replace all files on a pico) I'm generating a uf2 file with the littlefs following this  https://github.com/Gadgetoid/dir2uf2
I can create the uf2 file it sems that is getting flashed (is wiping my old py scripts ) but the new files are refusing to show up.
Is it possible what I'm trying to achieve? (I'm a hardware engineer with some sw knowledge)
Thanks,
A

**Comment by Josverl** [comment]
Likely your uf2 is not using the correct location or size for the filesystem for that port and board

I've not tried to just flash the file system, but the results appear the same as for a corrupted file system.

I suggest to check these with https://github.com/Josverl/vfs_merge

**Comment by adaptiveaurash** [comment]
The solution you suggested is actually better! Much appreciated, I'll give it a go.
A
